### PR TITLE
Tarantula script.

### PR DESCRIPTION
**Pull request by Dartellum**

### CODE REVIEW

File: profiles/base.yaml [hunk 1]
```diff
@@ -1144,6 +1144,12 @@ grave_junk:
 - millipede
 - cockroach
 
+#HE tarantula gift, list the skills you want the tarantula to use
+spider_noun: bio tarantula
+crafting_skills:
```
Comment (rpherbig) [inline profiles/base.yaml]: ❓ `*_skills` is really generic. What do you think about `spider_*_skills`?

File: tarantula.lic [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  crafting_skills += settings.crafting_skills
+  hunting_skills += settings.hunting_skills
+  outdoor_skills += settings.outdoor_skills
+
+  loop do
+    if Script.running?('workorders') || Script.running?('craft')
+      crafting_skills.each { |e| DRC.bput("turn my bio tarantula to #{e}", /You fiddle/, /With a/)
+      case DRC.bput("rub my #{spider_noun}", /The tarantula comes alive/, /It is mere/, /you currently aren\'t/, "It needs approximately \d+ (?:roisan|roisaen)", /You try, but it does nothing./)
+      when /It needs approximately (\d+) (?:roisan|roisaen)/
+        ptime = Regexp.last_match(1).to_i
```
Comment (rpherbig) [inline tarantula.lic]: Could you make this (and the next) variable names clearer?

Comment (Dartellum) [reply tarantula.lic]: Sure - ptime = pause time.

File: profiles/base.yaml (hunk 1 above)
Comment (Dartellum) [reply profiles/base.yaml]: Sure can. I am testing some other pieces and why I have lag.

**Comment by DryIce41** [comment]
Thank you for putting this together Dart.

I set it up and tried it a bit. 
During hunting-buddy it would occasionally use my set noun of Seven-legged tarantula. And sometimes it would try to use "bio tarantula"  

Also you set it up so you can choose scripts that will pause it. Athletics for example.

**Comment by Dartellum** [comment]
> Thank you for putting this together Dart.
> 
> I set it up and tried it a bit.
> During hunting-buddy it would occasionally use my set noun of Seven-legged tarantula. And sometimes it would try to use "bio tarantula"
> 
> Also you set it up so you can choose scripts that will pause it. Athletics for example. I called that as a before.

I had not changed a piece of code for the noun. I have that staged. As for the pausing, I did not add anything to pause other scripts, unless you are asking for that.

**Comment by DryIce41** [comment]
Not asking it to pause scripts, but something similar to walkingastro. Where you can designate scripts for it to not be used during.  Does it only cycle between the first two variable skills you tell it to?

**Comment by Dartellum** [comment]
Closing as another version is getting put up.

**Pull request closed without merging**